lms/templates/registration/activation_invalid.html

```diff
@@ -13,16 +13,16 @@
     <h1 class="invalid">${_("Activation Invalid")}</h1>
     <hr class="horizontal-divider">
 
-    <p>${Text(_('Something went wrong. Check to make sure the URL you went to was '
-                'correct -- e-mail programs will sometimes split it into two '
-                'lines. If you still have issues, e-mail us to let us know what happened '
+    <p>${Text(_('Something went wrong. Email programs sometimes split URLs into '
+                'two lines, so make sure the URL you're using is formatted '
```

Comment:
@srpearce: This is the second parse error reported as:
```
lms/templates/registration/activation_invalid.html: 16:8: mako-unparseable-expression:     <p>${Text(_('Something went wrong. Email programs sometimes split URLs into '
```

The problem here is that you introduced the word `you're` with a single quote, and the strings themselves are contained in single quotes.  You could change the starting and ending quote on lines 16-19 from a single quote to a double quote.